locales/en-US/addon.properties

```diff
@@ -18,7 +18,8 @@ add_to_queue= Add to queue
 clear= Clear
 history= History
 play_queue= Play Queue
-track_added_notification= Track added to queue
+track_added_notification= Item added to queue
```

Comment:
@proman21 I would suggest `item_added_notification`, but then this will need to be updated in `lib/get-locale-strings.js`